index.html

```diff
@@ -1866,6 +1864,9 @@ <h2 class="heading settled" data-level="2" id="constructs"><span class="secno">2
   or something similar to map the provided name to a string that it
   can store.</p>
    </aside>
+   <p>A <dfn class="dfn-paneled" data-dfn-type="dfn" data-noexport="" id="sorted-list">sorted list</dfn> is a list containing strings
+sorted in ascending order by code unit.</p>
```

Comment:
Looking at the ED, this seems to use a "forward definition". To simplify this, I'd probably move this paragraph below the note about names. That way you gain the discussion of DOMString's 16-bit code unit container goodness and don't have to repeat yourself here. Otherwise I'd add "16-bit" to "code unit"?